code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -859,5 +859,36 @@ GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
 	if(AI.eyeobj)
 		AI.eyeobj.relay_speech = TRUE
 
+
+//Fake Alert: Overloads a random number of lights across the station. Three uses.
+/datum/AI_Module/small/fake_alert
+	module_name = "Fake Alert"
+	mod_pick_name = "fake_alert"
+	description = "Assess the most probable threats to the station, and send a distracting fake alert by hijacking the station's alert and threat identification systems."
+	cost = 20
+	power_type = /datum/action/innate/ai/fake_alert
+	unlock_text = "<span class='notice'>You gain control of the station's alert system.</span>"
+	unlock_sound = "sparks"
+
+/datum/action/innate/ai/fake_alert
+	name = "Fake Alert"
+	desc = "Scare the crew with a fake alert."
+	button_icon_state = "fake_alert"
+	uses = 1
+
+/datum/action/innate/ai/fake_alert/Activate()
+	var/list/events_to_chose = list()
+	for(var/datum/round_event_control/E in SSevents.control)
+		if(!E.can_malf_fake_alert)
+			continue
+		events_to_chose[E.name] = E
+	var/datum/round_event_control/event_control = events_to_chose[input(owner,"Send fake alert","Fake Alert") in events_to_chose]
```

Comment:
Pretty sure this will runtime if the input returns null

Reply:
You're right. Good call.